packages/basura/lib/src/typography/basura_text_theme.dart

```diff
@@ -72,10 +79,9 @@ class BasuraTextThemeData extends Equatable {
   /// A text style for card subheadings.
   final TextStyle cardSubheading;
 
+  /// A text style for the github link.
+  final TextStyle gitHubLink;
```

Comment:
after doing this i have since realised it is probs overkill for a non reusable style. Happy to remove and just do inline

Reply:
Inline sounds good to me!